eea/volto/policy/profiles/default/catalog.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object meta_type="Plone Catalog Tool" name="portal_catalog">
+  <column value="image_scales" />
```

Comment:
@nileshgulia1 For this to work in Plone 5 we may also need the `image_scales` adapter from Plone 6: See https://github.com/plone/Products.CMFPlone/tree/master/Products/CMFPlone/image_scales

Reply:
Plone 5 compatibility added in [a04969d](https://github.com/eea/eea.volto.policy/pull/14/commits/a04969d024f924a3b2424048dc9b46c2d679ef2a)